src/Token/NewToken.tsx

```diff
@@ -0,0 +1,177 @@
+import {CSSObject} from '@styled-system/css'
```

Comment:
I think you should move this to the `drafts` directory and call it `IssueLabelToken` since that seems to be the component this would replace

Reply:
https://github.com/primer/react/tree/main/src/drafts